app/Http/Controllers/UserController.php

```diff
@@ -161,6 +161,10 @@ public function update(Request $request, $id)
             $user->external_auth_id = $request->get('external_auth_id');
         }
 
+        if ($request->filled('email') !== $request->old('email')) {
```

Comment:
`$request->filled('email')` does not get the new email but is a boolean if it's in the request data.
`$request->old('email')` is not garunted to be the users old email address, Only old input.

This line doesn't reliably check if the email is being changed. This can easily be amended though to work as expected.